Makefile

```diff
@@ -13,6 +13,7 @@ ETCDIR ?= ${DESTDIR}/etc
 ETCDIR_CRIO ?= ${ETCDIR}/crio
 BUILDTAGS ?= selinux seccomp $(shell hack/btrfs_tag.sh) $(shell hack/libdm_tag.sh)
 BASHINSTALLDIR=${PREFIX}/share/bash-completion/completions
+SELINUXOPT ?= $(shell  test -x /usr/sbin/selinuxenabled && selinuxenabled && echo -Z)
```

Comment:
Showing my Makefile ignorance here.  Does this set SELINUXOPT to '-Z' if /usr/sbin/selinuxenabled and selinuxenabled is present?  Otherwise SELINUXOPT is just set to ''?

Reply:
If SELinuxenabled exists and returns true indicating selinux is enabled then we will set the -Z flag.

selinuxenabled exist with 0 if enabled and 1 if not enabled.

Reply:
Thanks for the 411 and a neat Makefile trick.